doc/NPCs.md

```diff
@@ -228,21 +263,53 @@ The dynamic line is chosen based on whether the player character is performing a
 }
 ```
 
+### Based on the days since the Cataclysm
+The dynamic line is chosen based on the specified number of days have elapsed since the start of the Catacylsm.  Both entries are optional.  The line from `yes` will be shown if the that many days have passed, otherwise the line from `no`.
+
+```C++
+{
+    "days_since_cataclysm": "30",
+    "yes": "Things are really getting bad!",
+    "no": "I'm sure the government will restore services soon."
+}
+```
+
+### Based on the season
+The dynamic line is chosen is the current season matches `is_season`.  Valid seasons are "spring", "summer", "autumn", or "winter". The line from `yes` will be shown if the current season matches `is_season`, otherwise the line from `no`.
+```C++
+{
+    "is_season": "summer",
+    "yes": "At least it's a dry heat.",
+    "no": "What's going on?"
+}
+```
+
+### Based on the day/night cycle
+The dynamic line is chosen based whether it is currently daytime or nighttime.  Both entries must exist.
+
+```C++
+{
+    "is_day": "Sure is bright out.",
+    "is_night": "Look at the moon."
```

Comment:
Guess we need a phase of the moon condition too ;).

Reply:
Hasn't been requested, not doing it now because setting up the test case would be a headache.

Reply:
Sorry if it wasn't clear, that was a joke.